android/src/main/java/com/reactnativebledidcomm/peripheral/PeripheralManager.kt

```diff
@@ -62,6 +62,14 @@ class PeripheralManager(
         gattServer.addService(this.service)
     }
 
+    fun shutdownPeripheral() {
+        this.stopAdvertising()
```

Comment:
Do we not need to do a check here like with `shutdownCentral`?

Reply:
hmm thats strange - yes we do - and I did. probs forgot to commit that buffer, sry. will change it

Reply:
hmm thats strange - yes we do - and I did. probs forgot to commit that buffer, sry. will change it